### PR TITLE
Add the default item in the Schedule App

### PR DESCRIPTION
**Pull request by singa2000**
[Issue] N/A
[Problem] There is blank screen in the Schedule App
[Solution] Add the default item in the schedule app

Signed-off-by: singa2000 <cookie@samsung.com>

### CODE REVIEW

**Comment by TomaszLukawskiSam** [comment]
I think this is some regression... please check commit 97dc15549.

**Review by TomaszLukawskiSam — COMMENTED** [review]
Instead of your change,
you can add `type: "onoff"`
inside:
```
		defaultStore = [{
			on: tempDate.getTime(),
			off: tempDate.getTime() + 3900 * 1000, // + 1h 5m
                        type: "onoff",
			dayOfWeek: [1, 2, 3]
		}],
```

**Comment by singa2000** [comment]
@TomaszLukawskiSam 
In the https://github.com/Samsung/TAU/pull/1222 , it removed the default items.
So ,I think it's better to put the default item by "12:00 PM".
But I think https://github.com/Samsung/TAU/commit/97dc15549221088a7a458275bd92c54a4c721855 is better solution.

**Comment by TomaszLukawskiSam** [comment]
> @TomaszLukawskiSam
> In the #1222 , it removed the default items.
> So ,I think it's better to put the default item by "12:00 PM".
> But I think [97dc155](https://github.com/Samsung/TAU/commit/97dc15549221088a7a458275bd92c54a4c721855) is better solution.

I think you can reach the same effect by:
```
		defaultStore = [{
			on: tempDate.setHours(12, 0, 0),
			type: "on",
			dayOfWeek: [1, 2, 3]
		}],
```

File: examples/mobile/ScheduleApp/js/app.js [hunk 1]
```diff
@@ -131,7 +131,33 @@
...
+				<div class="ui-li-action action-container">
+					<day-indicator active="1"></day-indicator>
+					<select class="ui-on-off-switch">
+						<option value="off"></option>
+						<option value="on" selected></option>
+					</select>
+				</div>
+			</div>
+			<div class="content-footer">
+			Cool / 18°C / High
```
Comment (DohyungLim) [inline examples/mobile/ScheduleApp/js/app.js:156]: (This is a non-critical point.)
check indent.

**Review by DohyungLim — COMMENTED** [review]
1 comment.

**Review by DohyungLim — COMMENTED** [review]
check out Tomasz comment. and check for CI errors.

**Comment by singa2000** [comment]
It apply by https://github.com/Samsung/TAU/pull/1252.
So I close the PR

**Pull request closed without merging**